### PR TITLE
Database updates

### PR DESCRIPTION
**Pull request by iPhatty**
- added dummy data
- rename tables in plural (singular were causing issues)
- create db_connection file with database parameters

Relates #2 & #3

### CODE REVIEW

File: src/model/database/db_build.sql [hunk 1]
```diff
@@ -10,52 +10,75 @@ CREATE TABLE user
...
-    challenge_id BIGINT REFERENCES challenge(id),
+    user_id BIGINT REFERENCES users(id),
+    challenges_id BIGINT REFERENCES challenges(id),
     body VARCHAR(500) NOT NULL,
     added TIMESTAMPTZ NOT NULL DEFAULT NOW()
-);
\ No newline at end of file
+);
+
+INSERT INTO users (username, password, email) VALUES ('Tinky Wiky', 'password123', 'tinky@winky.com'),
```
Comment (haydnba) [inline src/model/database/db_build.sql]: user should be 'Tinky Winky' not 'wiky'

**Review by haydnba — CHANGES_REQUESTED** [review]
All good but please update your commenting in db_connection to be compatible with ESLint

**Review by i2xzy — APPROVED** [review]

File: src/model/database/db_build.sql (hunk 1 above)
Comment (iPhatty) [reply src/model/database/db_build.sql]: Thanks for catching the error! 👍

**Review by haydnba — APPROVED** [review]

**Review by n1c0z — APPROVED** [review]
:100: :smiley_cat:

**Comment by haydnba** [comment]
lovely changes Phat 🐚

**Pull request merged**